example/deployment/bash-scripts/setScriptPath.sh

```diff
@@ -1,9 +1,9 @@
-#!/usr/bin/env bash
+#!/bin/bash
 # sets up the different paths for calling deployment scripts
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-# Find example directory and normalise
-export EXAMPLE="$(cd "$DIR/../.." >/dev/null && pwd -P)"
+export EXAMPLE="$DIR/../.."
 export DOCKERBASHSCRIPTS="$EXAMPLE/deployment/local-docker/bash-scripts"
```

Comment:
The previous export EXAMPLE line is better in that it normalises symlinks and converts to an absolute path, the /dev/null redirection is to avoid errors in the case that cdspell shell option has been enabled in the shell. Please can this be reverted to the previous version?